### PR TITLE
CMM 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/chandra_legacy_of_fire.txt [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Name:Chandra, Legacy of Fire
+ManaCost:4 R
+Types:Legendary Planeswalker Chandra
+Loyalty:3
+T:Mode$ Phase | Phase$ End of Turn | TriggerZones$ Battlefield | Execute$ TrigDealDamage | TriggerDescription$ At the beginning of your end step, CARDNAME deals X damage to each opponent, where X is the number of planeswalkers you control.
+SVar:TrigDealDamage:DB$ DealDamage | Defined$ Opponent | NumDmg$ X
+SVar:X:Count$Valid Planeswalker.YouCtrl
+A:AB$ Mana | Cost$ AddCounter<1/LOYALTY> | Produced$ R | Amount$ X | Planeswalker$ True | SpellDescription$ Add a {R} each planeswalker you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/chandra_legacy_of_fire.txt:8]: fix Description

File: forge-gui/res/cardsfolder/upcoming/chandra_legacy_of_fire.txt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+A:AB$ RemoveCounter | Cost$ AddCounter<0/LOYALTY> | Choices$ Permanent.YouCtrl | Amount$ X | CounterType$ LOYALTY | CounterNum$ 1 | ChoiceOptional$ True | Planeswalker$ True | SubAbility$ DBExile | RememberAmount$ True | SpellDescription$ Remove a loyalty counter from each of any number of permanents you control. Exile that many cards from the top of your library. You may play them this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/chandra_legacy_of_fire.txt:9]: Amount$ doesn't exist

File: forge-gui/res/cardsfolder/upcoming/gatewatch_beacon.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Gatewatch Beacon
+ManaCost:2 W
+Types:Artifact
+K:ETBReplacement:Other:EntersTapped
+SVar:EntersTapped:DB$ Tap | Defined$ Self | ETB$ True | SubAbility$ DBAddCounter | SpellDescription$ CARDNAME enters the battlefield with three loyalty counters on it.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/gatewatch_beacon.txt:5]: wrong

File: forge-gui/res/cardsfolder/upcoming/vronos_masked_inquisitor.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
+Name:Vronos, Masked Inquisitor
+ManaCost:3 U U
+Types:Legendary Planeswalker Vronos
+Loyalty:5
+A:AB$ Pump | Cost$ AddCounter<1/LOYALTY> | ValidTgts$ Planeswalker.YouCtrl+Other | TargetMin$ 0 | TargetMax$ 2 | Planeswalker$ True | TgtPrompt$ Select up to two target other planeswalkers you control | SubAbility$ DelayedTrig | Planeswalker$ True | SpellDescription$ Up to two other target planeswalkers you control phase out at the beginning of the next end step. (Treat them and anything attached to them as though they don't exist until your next turn.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vronos_masked_inquisitor.txt:5]: Planeswalker$ twice

File: forge-gui/res/cardsfolder/upcoming/vronos_masked_inquisitor.txt [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+SVar:TrigPhase:DB$ Phases | Defined$ DelayTriggerRememberedLKI
+A:AB$ ChangeZone | Origin$ Battlefield | Cost$ SubCounter<2/LOYALTY> | Destination$ Hand | Planeswalker$ True | ValidTgts$ Permanent.nonLand+OppCtrl | TargetMin$ 0 | TargetMax$ OneEach | TargetsWithDifferentControllers$ True | TgtPrompt$ Select up to one target nonland permanent each opponent controls | SpellDescription$ For each opponent, return up to one target nonland permanent that player controls to its owner's hand.
+SVar:OneEach:PlayerCountOpponents$Amount 
+A:AB$ Animate | ValidTgts$ Artifact.YouCtrl | Cost$ SubCounter<7/LOYALTY> | Duration$ Permanent | TgtPrompt$ Select target artifact you control | Planeswalker$ True | Power$ 9 | Toughness$ 9 | Keywords$ Vigilance & Indestructible | staticAbilities$ Unblockable | Types$ Artifact,Creature,Construct | SpellDescription$ Target artifact you control becomes a 9/9 Construct artifact creature and gains vigilance, indestructible, and "This creature can't be blocked."
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/vronos_masked_inquisitor.txt:10]: RemoveCreatureTypes$

File: forge-gui/res/cardsfolder/upcoming/gatewatch_beacon.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
+Name:Gatewatch Beacon
+ManaCost:2 W
+Types:Artifact
+K:ETBReplacement:Other:DBAddCounter
+SVar:DBAddCounter:DB$ PutCounter | ETB$ True | CounterType$ LOYALTY | CounterNum$ 3
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/gatewatch_beacon.txt:5]: description?

File: forge-gui/res/cardsfolder/upcoming/gatewatch_beacon.txt [hunk 7]
```diff
@@ -2,10 +2,10 @@ Name:Gatewatch Beacon
 ManaCost:2 W
 Types:Artifact
 K:ETBReplacement:Other:DBAddCounter
-SVar:DBAddCounter:DB$ PutCounter | ETB$ True | CounterType$ LOYALTY | CounterNum$ 3
+SVar:DBAddCounter:DB$ PutCounter | ETB$ True | CounterType$ LOYALTY | CounterNum$ 3 | SpellDescription$ CARDNAME enters the battlefield with three loyalty counters on it.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/gatewatch_beacon.txt:5]: for now, why not use etb counter?

File: forge-gui/res/cardsfolder/upcoming/chandra_legacy_of_fire.txt [hunk 8]
```diff
@@ -0,0 +1,16 @@
+Name:Chandra, Legacy of Fire
+ManaCost:4 R
+Types:Legendary Planeswalker Chandra
+Loyalty:3
+T:Mode$ Phase | Phase$ End of Turn | TriggerZones$ Battlefield | Execute$ TrigDealDamage | TriggerDescription$ At the beginning of your end step, CARDNAME deals X damage to each opponent, where X is the number of planeswalkers you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/chandra_legacy_of_fire.txt:5]: "your" missing

**Review by tool4ever — APPROVED** [review]

**Pull request merged**